ptsprojects/zephyr/iutctl.py

```diff
@@ -149,6 +149,9 @@ def start(self, test_case):
 
         self.btp_socket.accept()
 
+        if self.board:
+            self.board.reset()
```

Comment:
@michal-narajowski FYI

Reply:
Why put reset here?

Reply:
Any questions?  I just think this is more appropriate, so we don’t need to add reset before call wait_for_ready_event.

Reply:
In addition, from a literal point of view, the start function already represents the start of the board.

Reply:
Ok, it doesn't really matter if reset is called in start or separately. This change only adds board.reset to start without other features of reset function.

Reply:
@michal-narajowski FYI